lib/spack/spack/fetch_strategy.py

```diff
@@ -1144,7 +1191,8 @@ def from_list_url(pkg):
     """If a package provides a URL which lists URLs for resources by
        version, this can can create a fetcher for a URL discovered for
        the specified package's version."""
-    if pkg.list_url:
+
+    if pkg.list_url is not None:
```

Comment:
Is there a distinction between `None` and `""` that was resulting in an error without updating this check?

Reply:
NTIR.  The attribute is set to `None` if not provided.  Otherwise, which is the case for all builtin packages (except flake8), it is a URL.  So I was following the "best practice" of checking for `None`.

Removed check.